src/bitcoin/utils.ts

```diff
@@ -0,0 +1,47 @@
+/* From @relayx/crypto src/bitcoin/utils.ts */
+function trim(buf: Buffer, natlen: number) {
+  return buf.slice(natlen - buf.length, buf.length)
+}
+
+/* From @relayx/crypto src/bitcoin/utils.ts */
+function pad(buf: Buffer, natlen: number, size: number) {
+  const rbuf = Buffer.alloc(size)
+  for (let i = 0; i < buf.length; i++) {
+    rbuf[rbuf.length - 1 - i] = buf[buf.length - 1 - i]
+  }
+  for (let i = 0; i < size - natlen; i++) {
+    rbuf[i] = 0
+  }
+  return rbuf
+}
+
+/* From @relayx/crypto src/bitcoin/utils.ts */
+export function BNToBuffer(bn: bigint, opts?: { size: number }) {
```

Comment:
import {etc} from '@noble/secp256k'

etc.numberToBytesBE

Reply:
updated in https://github.com/pow-co/relay-browser-extension/pull/16/commits/b2db333da31864dcc070bd81e41d2f6aeed65153